discord/opus.py

```diff
@@ -172,7 +179,7 @@ def _err_ne(result, func, args):
         [ctypes.c_char_p, ctypes.c_int], ctypes.c_int, _err_lt),
 ]
 
-def libopus_loader(name):
+def libopus_loader(name: str) -> ctypes.CDLL:
```

Comment:
```suggestion
def libopus_loader(name: str) -> Any:
```

Instead of `# type: ignore`ing everything this is the better way to go about doing it since the type checking is unnecessary.